source/crud/tests/v2/unacknowledged-replaceOne-hint-clientError.json

```
@@ -0,0 +1,104 @@
+{
+  "runOn": [
+    {
+      "maxServerVersion": "4.1.9"
+    }
+  ],
+  "data": [
+    {
+      "_id": 1,
+      "x": 11
+    },
+    {
+      "_id": 2,
+      "x": 22
+    }
+  ],
+  "collection_name": "ReplaceOne_hint",
+  "tests": [
+    {
+      "description": "Unacknowledged ReplaceOne with hint string fails with client-side error on server < 4.2",
+      "operations": [
+        {
+          "object": "collection",
+          "collectionOptions": {
+            "writeConcern": {
+              "w": 0
+            }
+          },
+          "name": "replaceOne",
+          "arguments": {
+            "filter": {
+              "_id": {
+                "$gt": 1
+              }
+            },
+            "replacement": {
+              "x": 111
+            },
+            "hint": "_id_"
+          },
+          "error": true
+        }
+      ],
+      "expectations": {},
+      "outcome": {
+        "collection": {
+          "data": [
+            {
+              "_id": 1,
+              "x": 11
+            },
+            {
+              "_id": 2,
+              "x": 22
+            }
+          ]
+        }
+      }
+    },
+    {
+      "description": "Unacknowledged ReplaceOne with hint document fails with client-side error on server < 4.2",
+      "operations": [
+        {
+          "object": "collection",
+          "collectionOptions": {
+            "writeConcern": {
+              "w": 0
+            }
+          },
+          "name": "updateOne",
```

This took me way too long to find...
```suggestion
          "name": "replaceOne",
```

---

Whoah good catch! Thank you 🙏